person.rb

```diff
@@ -31,4 +32,13 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def rent_book(book, date)
+    rental = Rental.new(self, book, date)
+    book.add_rental(rental)
+  end
```

Comment:
- [ ] The `Rental` class is the intermediate class between the `Book` class and the `Person` Class. This implies that the `Book` and `Person` classes have a **_many-to-many_** relationship through the `Rental` class. **A book can be rented by many people and a person can rent many books**. To implement this, you need to create a new `Rental` class whenever you want to a `Book` or `Person` object wants to perform a rental. Kindly implement this association correctly. 👍 💻 

#### Hint

```ruby
def add_rental(book, date)
  Rental.new(date, book, self)
end
```